README.md

```diff
@@ -41,6 +41,9 @@ Optional TLS configuration variables:
 | TEMPORAL_TLS_ENABLE_HOST_VERIFICATION | Enables verification of the server certificate                      | true    |
 | TEMPORAL_TLS_SERVER_NAME              | Target server that is used for TLS host verification                |         |
 | TEMPORAL_TLS_REFRESH_INTERVAL         | How often to refresh TLS Certs, seconds                             | 0       |
+| TEMPORAL_WEB_USE_HTTPS                | Enables HTTPS when viewing the temporal web UI in a browser         | false   |
```

Comment:
do you want exactly a separate env variable to enable/disable HTTPS? I'm thinking to rather rely having the cert set

Reply:
sure, I can do that.